x/icqoracle/types/keys.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import fmt "fmt"
+
+const (
+	ModuleName = "icqoracle"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+
+	// RouterKey defines the routing key
+	RouterKey = ModuleName
+)
+
+var (
+	ParamsKey        = []byte("params")
+	PriceQueryPrefix = []byte("pricequery")
+)
+
+func TokenPriceQueryKey(baseDenom, quoteDenom, poolId string) []byte {
+	return []byte(fmt.Sprintf("%s|%s|%s", baseDenom, quoteDenom, poolId))
+}
```

Comment:
should this be `TokenPriceKey` instead to line up with the keepers and struct type?